### PR TITLE
cl-cffi-gtk: disable tests

### PR DESCRIPTION
**Pull request by catap**
###### Type(s)

- [x] bugfix

###### Tested on

macOS 12.6.8 21G725 x86_64
Xcode 14.2 14C18

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint --nitpick`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?

### CODE REVIEW

File: lisp/cl-cffi-gtk/Portfile [hunk 1]
```diff
@@ -21,6 +21,7 @@ description             A Lisp binding to GTK 3
 long_description        {*}${description}
 
 # test system requires X11, maybe use Xvfb one day?
+test.run                no
```
Comment (reneeotten) [inline lisp/cl-cffi-gtk/Portfile:24]: why is this needed; MacPorts' default is set like this, unless you have it enabled by default in the lisp PG?

Comment (catap) [reply lisp/cl-cffi-gtk/Portfile:24]: It is enabled in lisp PG because almost all ports have working tests :)
